apps/i18n/dance/en_us.json

```diff
@@ -4,14 +4,17 @@
   "danceAiFeedbackAiBlockNotConnectedSetup": "⚠️ Connect the `A.I. generate effect` block to the `setup` block.",
   "danceAiFeedbackAiBlockNoParametersOnEvent": "⚠️ The effect didn't change after 2 measures. Choose three emojis for A.I. to generate the effect.",
   "danceAiFeedbackConnectPressUpArrowEvent": "⚠️ Press the up arrow ▲ button after pressing *\"Run\"*.",
-  "danceAiFeedbackCustomize": "⚠️ You need to change the effect A.I. generated. You can either regenerate the effects, customize the code A.I. created, or start over with new emojis.",
+  "danceAiFeedbackCustomize": "⚠️ You need to change the effect A.I. generates. You can either regenerate the effect, customize the code A.I. writes, or start over with new emojis.",
   "danceAiFeedbackEffectChangeAfterMeasures": "⚠️ Connect the `A.I. generate effect` block to the `after 2 Measures` event.",
   "danceAiFeedbackKeyPress": "⚠️ You pressed a key, but your dancer didn't do a new move.",
-  "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press at least one of the arrow keys while your dance is running.",
-  "danceAiFeedbackMatchDancer": "⚠️ You must have the same dancer selected in the make new dancer block and the dancer do new move block.",
+  "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press the right or left arrow while your dance is running.",
+  "danceAiFeedbackMatchDancer": "⚠️ You must have the same dancer selected in the `make a new dancer` block and the `dancers do dance move` block.",
   "danceAiFeedbackNeedNewDancer": "⚠️ You need to make a dancer.",
   "danceAiFeedbackNewMove": "⚠️ You need to add a block to make the dancer do a new move.",
   "danceAiFeedbackNoAiBlock": "⚠️ Add the `A.I. generate effect` block to setup.",
+  "danceAiFeedbackNoNewMove": "⚠️ Your dancer didn't do a new move. Keep trying!",
+  "danceAiFeedbackNoNewMoveDetailed": "⚠️ Your dancer didn't do a new move. Check the following:\n- the same dancer must be selected in the `make a new dancer` block and the `dancers do dance move` block\n- the `make a new dancer` block must be placed before the `dancers do dance move` block.",
```

Comment:
This is far too long for a validation string, especially in the first level of HOC.  Furthermore, our validation should figure out what's wrong and give concise, specific feedback, rather than putting the burden on the student to go through a checklist.